src/config/docs.md

```diff
@@ -37,6 +37,8 @@ This page is automatically generated by `./risedev generate-example-config`
 | full_gc_interval_sec | Interval of automatic hummock full GC. | 86400 |
 | hummock_version_checkpoint_interval_sec | Interval of hummock version checkpoint. | 30 |
 | hybird_partition_vnode_count |  | 4 |
+| hybrid_few_partition_threshold |  | 134217728 |
+| hybrid_more_partition_threshold |  | 536870912 |
```

Comment:
I believe no one will understand what these two configs mean by looking at the names. 
How about:
`compact_task_table_size_split_threshold_low`
`compact_task_table_size_split_threshold_high`

Also, let's fill-in the description colum here in `docks.md` to explain what these two configs mean.

Reply:
how about `compact_task_table_size_partition_threshold_low `